### ISSUE TITLE
All supervisor log output became green

### ISSUE THREAD

**Issue by frenck** [home-assistant/home-assistant-polymer#4625]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

All logout became green:

![image](https://user-images.githubusercontent.com/195327/73221055-37612280-4160-11ea-9a7a-f70906ab6e4b.png)

## Expected behavior

Old panel:

![image](https://user-images.githubusercontent.com/195327/73221070-434ce480-4160-11ea-9926-e1a39620afdd.png)

## Steps to reproduce

Open up the supervisor logs.

## Environment

- Home Assistant release with the issue: 0.104.3 / Supervisor 196 (dev)
- Last working Home Assistant release (if known): Supervisor 195
- UI Type (States or Lovelace): n/a
- Browser and browser version: Version 79.0.3945.117 (Official Build) (64-bit)
- Operating system: macOS Mojave

## Problem-relevant configuration

```yaml
n/a
```

## Javascript errors shown in your browser console/inspector

```txt
Nothing.
```

**Comment by pvizeli** [comment]
I don't see that as an issue by the way. If I have a vote, I would vote to led it green

**Comment by ludeeus** [comment]
I also vote green.
I can not remember changing anything related to this.
Isn't the color set by the supervisor backend?

**Comment by frenck** [comment]
Could be? ANSI support maybe wasn't in there before?

I just noticed a difference in behavior and raised it.

However, I do think the green is harder to read, also, consider people with a color vision deficiency, who can't really separate green from red in general.

If you guys agree this is correct and wished for, well, we should close this issue 👍

**Comment by pvizeli** [comment]
Yes, the frontend had a hack for this but you are right, the color is correct to handle by logger on the backend

**Comment by pvizeli** [comment]
I removed the bug label because we can discuss it. However, with the upcoming changes and integration, maybe this discussion is useless.

**Comment by frenck** [comment]
Yeah the bug label is automatically added in this repo 🤷‍♂

**Comment by pvizeli** [comment]
@frenck feel free to make a PR to supervisor repo to change the color for the info to not green. I will merge it. But we should not include the ASCII filter for colors on UI again.

**Comment by frenck** [comment]
To add some statistics (WikiPedia):

> Red–green color blindness affects up to 8% of males and 0.5% of females of Northern European descent

Another source: <https://www.nei.nih.gov/learn-about-eye-health/eye-conditions-and-diseases/color-blindness>

**Comment by frenck** [comment]
Let's move this issue to the Supervisor repository 👍
